.github/workflows/build.yaml

```diff
@@ -1,6 +1,9 @@
-name: "Testing"
+name: "Mayastor tests"
```

Comment:
I think we also need to update the -- what's the name again? The guards for merge block/unblock. It's not called that but I think you know what I mean😅

Reply:
There we refer to job names that have not changed. So no change is required there 👍